apps/mdn/mdn-aws/infra/multi_region/tf/main.tf

```diff
@@ -9,3 +9,94 @@ terraform {
     region = "us-west-2"
   }
 }
+
+
+#########################################
+# EFS
+#########################################
+
+module "efs-dev" {
+    source = "efs"
+    efs_name = "dev"
+    subnets = "${var.subnets}"
+    nodes_security_group = "${var.nodes_security_group}"
+}
+
+#module "efs-stage" {
+#    source = "efs"
+#    efs_name = "stage"
+#    subnets = "${var.subnets}"
+#    nodes_security_group = "${var.nodes_security_group}"
+#}
+#
+#module "efs-prod" {
+#    source = "efs"
+#    efs_name = "prod"
+#    subnets = "${var.subnets}"
+#    nodes_security_group = "${var.nodes_security_group}"
+#}
+
+
+#########################################
+# Redis
+#########################################
+
+module "redis-dev" {
+    source = "redis"
+    redis_name = "dev"
+    redis_node_size = "cache.t2.micro"
+    redis_num_nodes = 1
+    subnets = "${var.subnets}"
+    nodes_security_group = "${var.nodes_security_group}"
+}
+
+#module "redis-stage" {
+#    source = "redis"
+#    redis_name = "stage"
+#    redis_node_size = "cache.t2.micro"
```

Comment:
per a discussion with @escattone, let's use 3 x `cache.t2.small` for staging and possibly dev